docs/en/index.md

```diff
@@ -89,6 +89,8 @@ class FooListener extends Nette\Object implements Kdyby\Events\Subscriber
 Now when I invoke the event `onStartup`, the listener method should get called
 
 ```php
+namespace Nette\Application;
```

Comment:
Oh I see. But this doesn't really have anything to do with Nette. Remove the namespace and use this:
`$this->evm->dispatchEvent('Application::onStartup', new EventArgsList(array($this)));`